setup-env/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Setup Environment'
+description: 'Setup dependencies for base container'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install updates and missing packages
+      run: |
+        export DEBIAN_FRONTEND=noninteractive
+        apt update && apt upgrade -y
+
+        # Install git if not exist
+        if ! command -v git &> /dev/null; then
+          apt install git -y
+        fi
+
+        # Install awscli if not exist
+        if ! command -v aws &> /dev/null; then
+          apt install awscli -y
+        fi

```

Comment:
Double-check: does `apt` have the latest versions of `git` and `awscli`? If not, would this be okay anyway?

Reply:
No, they are not the latest versions but they are supposed to be one of the stable versions:
https://packages.ubuntu.com/focal/awscli
https://packages.ubuntu.com/focal/git
Unless the CLI of them changes significantly. otherwise there should not be any problem.